lib/plugins/index.js

```diff
@@ -11,10 +14,19 @@ const _ = require('lodash');
  * @param  {Any} args
  */
 function executeHook(key, args) {
-  _.map(module.exports.plugins, (plugin) => {
-    if (_.isFunction(plugin[key])) {
+  _map(module.exports.plugins, (plugin) => {
```

Comment:
This should just be an `_.each` or `.forEach`

Reply:
Oh that is true, will update. I wonder why I had it there before...Hmmm